### ISSUE TITLE
Server uptime not displayed

### ISSUE THREAD

**Issue by j-ed** [nextcloud/serverinfo#159]
## Expected behavior
The uptime of a server should be displayed.

## Current behavior
The server uptime is not displayed.

![grafik](https://user-images.githubusercontent.com/2572764/67198321-081f7480-f3ff-11e9-9805-5b61fa57236b.png)

## Steps to reproduce
1. Open the systems app in the settings dialog.

I tried to find the root cause of the problem and identified an incorrect command line syntax causing the problem in `lib/OperatingSystems/DefaultOs.php`:

CURRENTLY USED COMMAND: `uptime -p`
WORKING COMMAND: `uptime`
```
> uptime --version
uptime (GNU coreutils) 8.25

> uptime
 12:18pm  up 54 days  0:02,  8 users,  load average: 1.48, 1.26, 1.11
```

#### Server Configuration
OS: Linux 3.16.70
Web server: Apache2 2.4.41
Database: MariaDB 10.2.24
PHP version: 7.3.10
Nextcloud version: 17.0.0
**Serverinfo app version: 1.7.0**

#### Client Configuration
Browser: Mozilla Firefox 69.0.3
Operating system: Windows 10

**Comment by tflidd** [comment]
Only some uptime programs have the `-p` option (it stands for pretty), so it looks like:

> up 7 weeks, 5 days, 16 hours, 30 minutes

So looks like, we should do something similar.

**Comment by Starbix** [comment]
OSs based on busybox such as Alpine Linux don't support the pretty flag.

Would it be possible to include a check whether `uptime` supports the `-p` flag and fall back to calling it without `-p` and maybe add some kind of regex to emulate the `-p` flag?

Right now even some official installations are affected, eg. `nextcloud:fpm-alpine`.

**Comment by kesselb** [comment]
Fixed by #183 for Nextcloud 19.
